### DISCUSSION TITLE
Show much more info on the status server

### DISCUSSION THREAD

**Discussion by NotAFile** [piqueserver/piqueserver#666] [Feature Requests]
For example, look at all the stuff aloha.pk shows: http://central.aloha.pk:34886/

What we show is very very bare in comparison.

We might need to be careful to still make it usable and compact, though.

**Comment by samuelallan72** [comment]
Internet archive link to the above status server page: https://web.archive.org/web/20181018013434/http://central.aloha.pk:34886/
